src/fsharp/fsc/fsc.fsproj

```diff
@@ -5,6 +5,7 @@
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <TargetFrameworks>net46;netcoreapp2.1</TargetFrameworks>
+    <TargetFrameworks Condition="'$(OS)' == 'Unix'">netcoreapp2.1</TargetFrameworks>
```

Comment:
>Condition="'$(OS)' == 'Unix' [](start = 22, length = 28)

Shouldn't need to do this. I am guessing you get an error finding Ref Assemblies otherwise? If that is so, you can set `UsingToolNetFrameworkReferenceAssemblies` to true in `/eng/Versions.props` like so:
[https://github.com/dotnet/roslyn/blob/master/eng/Versions.props#L283](https://github.com/dotnet/roslyn/blob/master/eng/Versions.props#L283)

That will allow you to target `net46` on any OS.
Also I think you might want to retarget to `net472` at some point.